tests/migration_test.go

```diff
@@ -872,71 +872,7 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 				}, 240*time.Second, 1*time.Second).Should(Succeed())
 			})
 		})
-		Context("with a shared ISCSI Filesystem PVC (using ISCSI IPv4 address)", func() {
-			BeforeEach(func() {
-				tests.BeforeTestCleanup()
-
-				if !tests.HasCDI() {
-					Skip("Skip DataVolume tests when CDI is not present")
-				}
-
-				// set unsafe migration flag
-				cfg := getCurrentKv()
-				unsafeMigrationOverride := true
-				cfg.MigrationConfiguration.UnsafeMigrationOverride = &unsafeMigrationOverride
-				tests.UpdateKubeVirtConfigValueAndWait(cfg)
-			})
-
-			It("[test_id:3238]should migrate a vmi with UNSAFE_MIGRATION flag set", func() {
-				// Normally, live migration with a shared volume that contains
-				// a non-clustered filesystem will be prevented for disk safety reasons.
-				// This test sets a UNSAFE_MIGRATION flag and a migration with an ext4 filesystem
-				// should succeed.
-
```

Comment:
can we replicate this test without iscsi? I'm unsure what the ramifications of removing this are.

@vladikr maybe you have thoughts about whether we need to continue testing "unsafe_migration"?

Reply:
Originally, we have introduced this at the time when libvirt considered CEPH as an unsafe filesystem for migrations, but we need ceph support.
At that time, prior to libvirt 5.6, the writeback, writethough, and unsafe disk cache modes were considered unsafe too. 

Nowadays, migrating with the above disk cache modes is considered safe afaik [1]. It's really difficult to find a scenario where this would be needed. 
I can only think about using a local PV with ext4, but we are protected anyway. We can force this, but it will be hard I think.

On the other hand, PR only removes the test, the flag will remain in place.

I didn't feel great about exposing this flag in the first place. I think we only introduced it because we didn't have a good build process for the libvirt image and we didn't know when or will it ever be fixed. 

[1]  https://github.com/libvirt/libvirt/blob/master/src/qemu/qemu_migration.c#L1484